### PR TITLE
feat: score effect update

### PR DESCRIPTION
**Pull request by ghost**
1. The score standard for changing effects has been changed from 800 points to 400 points.

2. Various score effects have been added.

As before, the color changes every 800 points.

### CODE REVIEW

**Review by NiceGuy1313 — APPROVED** [review]
Good. I think we'll be able to enjoy more fun games

File: src/engine/DrawManager.java [hunk 1]
```diff
@@ -589,9 +589,13 @@ public void scoreEmoji(final Screen screen, final int score) {
 			backBufferGraphics.setColor(scoreColor(800));
 			backBufferGraphics.drawString(" Z...z..    ( _ . _ )", screen.getWidth() - 250, 25);
 		}
-		if (score >= 800 && score < 1600) {
+		if (score >= 800 && score < 1200) {
 			backBufferGraphics.setColor(scoreColor(800));
```
Comment (jeongwoo903) [inline src/engine/DrawManager.java:593]: At first, I thought, if the conditional is determining the score range, why is it determining the score again inside `scoreColor()`? It's neat, but inefficient... But then I noticed that `drawScore()` is using `scoreColor()`. 
Looking at that code, I wondered if I need to use `scoreColor()` to manage the color separately? But if it's a necessary method anyway, it doesn't seem like a problem to fix 🤔.

(처음엔 조건문에서 score 범위를 판정해주는데, 왜 다시 `scoreColor()` 내부에서 score을 다시 판정하지? 깔끔하긴 한데 비효율 적이지 않나? 라고 생각했는데 `drawScore()`에서 `scoreColor()`을 쓰고 있더군요.. 
해당 코드만 보면 `scoreColor()`을 굳이 사용해서 color를 따로 관리할 필요가 있을까? 했는데 어짜피 필요한 매서드라면 안고쳐도 문제없어 보이긴 합니다 🤔)

**Review by jeongwoo903 — APPROVED** [review]

**Review by 258xsw — COMMENTED** [review]
If there is a correlation between emojis and scores, it will be a more interesting game!

**Comment by ghost** [comment]
Team members, thank you for the detailed review!

**Comment by ghost** [comment]
This PR was created before the auto-merge setting. Thank you.

**Pull request merged**